sass/_variables.scss

```diff
@@ -0,0 +1,18 @@
+/*- fonts -*/
+$main-font: Helvetica, Arial, "Lucida Grande", sans-serif;
+
+/*- colors -*/
+
+$alpha: #FF6124;
+$beta: #585050;
+$gamma: #989595;
+$omega: #703A20;
+$kappa: #C1530D;
```

Comment:
Not sure I follow your naming scheme, shouldn't the fourth and fifth variables be `$delta` and `$epsilon`? Abstract is one thing, but remember that variable names communicate meaning to other developers.